SlimApp/NewClient/NewClientView.swift

```diff
@@ -9,34 +9,69 @@ import SwiftUI
 
 struct NewClientView: View {
     @StateObject var viewModel: NewClientViewModel
+    let emojis = ["😀", "😎", "🥰", "😴", "🤔", "🤯"]
+    let decimalFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.allowsFloats = true
+        formatter.minimumFractionDigits = 0
+        formatter.maximumFractionDigits = 2
+        return formatter
+    }()
     var body: some View {
         NavigationView {
             // TODO: e.grishina rewrite it with ScrollView
             Form {
                 Section {
                     TextField("Name", text: $viewModel.name)
-                    TextField("Surname", text: $viewModel.surname)
-                    TextField("E-mail", text: $viewModel.email)
-                        .keyboardType(.emailAddress)
-                    TextField("Phone number", text: $viewModel.phoneNumber)
-                        .keyboardType(.phonePad)
+                    Picker("Select sex", selection: $viewModel.sex) {
+                        ForEach(Sex.allCases, id: \.self) { sex in
+                            Text(sex.rawValue)
+                        }
+                    }
+                    Picker("Select an avatar", selection: $viewModel.avatar) {
+                        ForEach(emojis, id: \.self) { emoji in
+                            Text(emoji)
+                        }
+                    }
                 }
                 Section {
-                    TextField("Height", text: $viewModel.height)
-                        .keyboardType(.decimalPad)
-                    TextField("Weight", text: $viewModel.weight)
-                        .keyboardType(.numberPad)
+                    HStack {
+                        Text("Weight (Kg)")
+                        TextField("0", value: $viewModel.weight, formatter: decimalFormatter)
+                            .keyboardType(.numbersAndPunctuation)
+                        Stepper("Insert Weight", value: $viewModel.weight, step: 0.5)
```

Comment:
negative values ​​should be limited

Reply:
also maybe it is better to provide base value which is closer to the average (not 0)

Reply:
After having a talk with about the stepper and due to some troubles implementing it with bmi and behaviour of the textfields, I decided to remove the stepper so now I think that having 0 value as default for the textfield should be ok